kalite/coachreports/static/js/coachreports/tabular_reports/hbtemplates/tabular-view.handlebars

```diff
@@ -1,7 +1,7 @@
 <div>
     <div class="row">
         <div class="col-md-6 col-xs-12">
-        <label id="legendMargin" for="legend">Key</label>
+        <label id="legendMargin" for="legend">{{_ "Color legend"}}</label>
```

Comment:
The word "Key" will confuse translators, that's why I replaced it - sadly it's very long, but it shows up okay.

A secondary issue is that the legend is shown twice which IMO is plain silly :)

![image](https://user-images.githubusercontent.com/374612/29475709-4108ca72-8461-11e7-9b9a-a99995b0b645.png)